sdk/python/kfp/_client.py

```diff
@@ -304,6 +304,25 @@ def create_experiment(self, name, description=None, namespace=None):
       IPython.display.display(IPython.display.HTML(html))
     return experiment
 
+  def get_pipeline_id(self, name, page_token='', page_size=100):
+    """Returns the pipeline id if a pipeline with the name exsists.
```

Comment:
Note, when multi user mode enabled, you can only query pipelines by namespace.

Reply:
Could you elaborate further on this @Bobgy? Do you mean I should also pass in the namespace as a filter? Or do you mean that the results will be limited to the namespace I have access to? I have checked the API and they don't mention the use of namespace as a parameter: https://www.kubeflow.org/docs/pipelines/reference/api/kubeflow-pipeline-api-spec/#tag-PipelineService